Tests/FoundationTests/URLExtensionsTests.swift

```diff
@@ -20,11 +20,16 @@ final class URLExtensionsTests: XCTestCase {
 
     func testAppendingQueryParameters() {
         XCTAssertEqual(url.appendingQueryParameters(params), queryUrl)
+		let replaceURL = URL(string: "https://www.google.com?q=swift%20swifter")!
+		XCTAssertEqual(replaceURL.appendingQueryParameters(params, replaceExisting: true), queryUrl)
     }
 
     func testAppendQueryParameters() {
         url.appendQueryParameters(params)
         XCTAssertEqual(url, queryUrl)
+		let replaceURL = URL(string: "https://www.google.com?q=swift%20swifter")!
+		replaceURL.appendQueryParameters(params, replaceExisting: true)
+		XCTAssertEqual(replaceURL, queryUrl)
```

Comment:
Can we add a test where there is an additional param added as well, only this `q` param makes function seem a no-op.

Reply:
Done